qa/automation/ScoringPortal.Tests.Scenarios/terenteva/sign_in_with_Google_account.feature

```diff
@@ -0,0 +1,91 @@
+	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	When User drags cursor to the 'Sign in with Google' button
+	Then 'Sign in with Google' button is highlighted
+	And Cursor appearance is changed from standard to Link pointer
+	And Hyperlink address is shown at the bottom left corner of the page
+
+ Scenario Outline: 1_02. User is redirected to Google Login page
+ 	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	And User Google account state is <User account state>
+	When User clicks on 'Sign in with Google' button
+	Then User is redirected to Google Accounts Login page
+	And Google Accounts Login page is opened in the same browser tab
+
+	  Examples: 
+	  | User account state                                     |
+	  | User has no Google account                             |
+	  | User has a Google account, is not signed in the account |
+	  | User has a Google account, is signed in it, not signed in QSPP before from the device and browser used |
+
+ Scenario: 1_03. User goes straight from QSPP 'User not signed in' to QSPP 'User signed in'
+	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	And User has a Google account
+	And User is signed in this Google account
+	And User has signed in to QSPP before from the device and browser used
+	When User clicks on 'Sign in with Google' button
+	Then User is redirected to QSPP 'User signed in' version
+	And QSPP 'User signed in' version is opened in the same browser tab
+
+ Scenario Outline: 1_04. User is signed in QSPP after managing Google account
+	Given QSPP 'User NOT signed in' version is opened in a browser tab
+	And User account state is <User account state>
+	And User has clicked on 'Sign in with Google' button on QSPP 'User not signed in'
+	And User was redirected to Google Accounts Login page
+	And Google Accounts Login page is opened in the browser tab
+	When User completes <Actions> according to <User account state>
+	Then User is redirected to QSPP 'User signed in' version
+	And QSPP 'User signed in' version is opened in the same browser tab
+
+	  Examples: 
+	  | User account state                                                     | Actions								 |
+	  | User has no Google account                                             | User registers an account and signs in  |
+	  | User has a Google account, is not signed in the account                | User signs in the account				 |
+	  | User has a Google account, is signed in it, not signed in QSPP before* | User chooses the account to proceed	 |
+	  # *from the device and browser used
+
+ Scenario: 1_05. User signs out from the QSPP
+	Given QSPP 'User signed in' version is opened in a browser tab
+	When User clicks Sign out button marked with sign out icon
+	Then User is redirected to QSPP 'User NOT signed in' version
+	And QSPP 'User NOT signed in' version is opened in the same browser tab
+
+ Scenario: 1_06. User stays signed in after QSPP 'User signed in' version is refreshed
```

Comment:
For me it is a case of 'user gets access to functionality when signed in', so that refreshing the page shouldn't affect this, maybe I'd better rephrase this Scenario name in that way, would that be ok with you?

Reply:
Do you think in this context "get access" is same as "sign in" or being "authenticated" as he is already signed in, remember authentication of user being signed in is because Google allows it, so it is more related on what is going on with Google account rather than simple page refresh as if you sign out from Google account on another page, refreshing the app will revoke access to app

Reply:
I meant here 'signed in to the system', maybe I'll change it that way? Or do you propose to delete this Scenario completely?
I'm not sure how to apply these terms here. @shepelevaolga Could you please comment on this matter?
 
Actually I tested signing out from Google off system and refreshing as in Scenario 1_07. and actually when I logged out from Google, I stayed signed in to the Scoring portal, so I suppose, there is not just Google involved.

Reply:
In my opinion scenarios 5-7 do not belong here and have to be deleted, and as for others you should really consider whether you are testing the requirement we have as to me it looks more like you are testing Google decisions. Let's see @shepelevaolga in this as otherwise I won't be able to approve and merge PR into the branch